ocs_ci/ocs/resources/pod.py

```diff
@@ -16,7 +16,8 @@
 from ocs_ci.ocs.ocp import OCP, verify_images_upgraded
 from tests import helpers
 from ocs_ci.ocs import workload
-from ocs_ci.ocs import constants, defaults, node
+from ocs_ci.ocs import constants, defaults
```

Comment:
Is this change related ? If not can it be moved to its own PR with issue?

Reply:
not needed, will revert